tests/e2e/__snapshots__/e2e.test.ts.snap

```diff
@@ -64,6 +73,10 @@ import * as PrismaClient from \\".prisma/client\\"
 import { core } from \\"nexus\\"
 declare global {
   interface NexusGenCustomInputMethods<TypeName extends string> {
+    /**
+     * The \`BigInt\` scalar type represents non-fractional signed whole numeric values.
```

Comment:
Let's improve this to include a link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt.